pkg/controller/drain.go

```diff
@@ -369,50 +368,57 @@ func filterPodsWithPv(pods []api.Pod) ([]*api.Pod, []*api.Pod) {
 	return podsWithPv, podsWithoutPv
 }
 
-func (o *DrainOptions) evictPods(pods []api.Pod, policyGroupVersion string, getPodFn func(namespace, name string) (*api.Pod, error), stopCh <-chan struct{}) error {
+func (o *DrainOptions) getTerminationGracePeriod(pod *api.Pod) time.Duration {
+	if pod == nil || pod.Spec.TerminationGracePeriodSeconds == nil {
+		return time.Duration(0)
+	}
+
+	return time.Duration(*pod.Spec.TerminationGracePeriodSeconds) * time.Second
+}
+
+func (o *DrainOptions) getGlobalTimeoutForPodsWithoutPV(pods []*api.Pod) time.Duration {
+	var tgpsMax time.Duration
+	for _, pod := range pods {
+		tgps := o.getTerminationGracePeriod(pod)
+		if tgps > tgpsMax {
+			tgpsMax = tgps
+		}
+	}
+
+	return time.Duration(tgpsMax)*time.Second + PodsWithuotPVDrainGracePeriod
```

Comment:
`getTerminationGracePeriod()` already returns the Period(int) multiplied by seconds right? We are again multiplying it here with seconds? A mistake I am guessing?

Reply:
Typo for without
```suggestion
	return time.Duration(tgpsMax)*time.Second + PodsWithoutPVDrainGracePeriod
```

Reply:
Fixed.